tests/FamilyHubs.ReferralApi.UnitTests/WhenUsingReferralCommands.cs

```diff
@@ -94,6 +93,61 @@ public async Task ThenGetReferralsByReferrer()
             result.Items[0].Should().BeEquivalentTo(testReferral);
         }
 
+        [Fact]
+        public async Task ThenGetReferralsByReferrerByTextSearching()
+        {
+            //Arange
+            var myProfile = new AutoMappingProfiles();
+            var configuration = new MapperConfiguration(cfg => cfg.AddProfile(myProfile));
+            IMapper mapper = new Mapper(configuration);
+            var logger = new Mock<ILogger<CreateReferralCommandHandler>>();
+            var mockApplicationDbContext = GetApplicationDbContext();
+            var testReferral = GetReferralDto();
+            CreateReferralCommand command = new(testReferral);
+            CreateReferralCommandHandler handler = new(mockApplicationDbContext, mapper, logger.Object);
+            await handler.Handle(command, new System.Threading.CancellationToken());
+
+            GetReferralsByReferrerCommand getcommand = new("Bob Referrer", 1, 10, "Robert", default!);
+            GetReferralsByReferrerCommandHandler gethandler = new(mockApplicationDbContext);
+
+
+            //Act
+            var result = await gethandler.Handle(getcommand, new System.Threading.CancellationToken());
+
+            //Assert
+            result.Should().NotBeNull();
+            result.Items[0].Should().BeEquivalentTo(testReferral);
```

Comment:
could you please add an assert about the new field that you have added so that we know this test specifically returns expected matching value back

Reply:
Done